### DISCUSSION TITLE
Inheritance of permission inside a thread?

### DISCUSSION THREAD

**Discussion by chaoxu** [edemaine/coauthor#596] [Q&A]
I have a group where everyone does not have read permission, only post permission.

This allows each person to create some thread, tag their coauthor, discuss, and no one else can see it. (apart from superuser). So it works like creating a private chat.

One thing I noticed:

1. Post a thread with authors X and coauthor Y.
2. Y replies to the thread, without adding X as coauthor.
3. X cannot see Y's reply.

This show the ability to read does not inherit in the thread. Is this the desired behavior?

**Comment by edemaine** [comment]
Interesting. It's intentional that coauthorship is not inherited; that's not what coauthorship is "meant for". But private messages have the behavior you intend, where you can explicitly list people who can see a message without being coauthors, and that list is initialized to everyone from the parent message. This is probably what you want, but it's currently not possible without private messages. Typically, a superuser would create a thread that has private messages allowed or required, and then replies will behave like you want. But there isn't a way to do this at a group level. I wonder if there's an interface we could build to make your setting work like private messages already do...
